src/iovledgerwallet.ts

```diff
@@ -86,16 +86,16 @@ function deserialize(data: WalletSerializationString): LedgerSimpleAddressWallet
 
 type IdentityId = string & As<"identity-id">;
 
-export class LedgerSimpleAddressWallet implements Wallet {
+export class IovLedgerWallet implements Wallet {
   public static readonly implementationId = "ledger-simpleaddress" as WalletImplementationIdString;
```

Comment:
Could you update the content of `IovLedgerWallet.implementationId` as well? It does not need to be something specific. Just a identifier such that the software knows which class to use when restoring a wallet from database. You can have a look at the values for `implementationId` for the other wallets in IOV-Core.